lib/src/utils/schema/adders.ts

```diff
@@ -89,7 +91,7 @@ function addWidget(
 function addSection(
   verna: VernaContextProps,
   id: string,
-  objectFieldTemplate: ObjectFieldTemplateType,
+  objectFieldTemplate: React.FunctionComponent<ObjectFieldTemplateProps>,
```

Comment:
SectionTemplate is maybe a better name since we do not use `objectFieldTemplate.section` ?